src/Command/CodeStudio/CodeStudioWizardCommand.php

```diff
@@ -163,9 +187,34 @@ private function createProjectAccessToken(array $project, string $projectAccessT
     return $projectAccessToken['token'];
   }
 
-  private function setGitLabCiCdVariables(array $project, string $cloudApplicationUuid, string $cloudKey, string $cloudSecret, string $projectAccessTokenName, string $projectAccessToken, string $phpVersion): void {
+  private function setGitLabCiCdVariablesForPhpProject(array $project, string $cloudApplicationUuid, string $cloudKey, string $cloudSecret, string $projectAccessTokenName, string $projectAccessToken, string $phpVersion): void {
+    $this->io->writeln("Setting GitLab CI/CD variables for {$project['path_with_namespace']}..");
+    $gitlabCicdVariables = CodeStudioCiCdVariables::getDefaultsForPhp($cloudApplicationUuid, $cloudKey, $cloudSecret, $projectAccessTokenName, $projectAccessToken, $phpVersion);
```

Comment:
Don't rely on classes from the system under test (e.g., CodeStudioCiCdVariables). These variables should be hardcoded in the test case or fixture.